### PR TITLE
Add component information in app and service lists

### LINKED ISSUE #764: Add application name and service binding class name to apps/instances listing
Currently, the deployed applications listing shows the name of the release and the icon, but does not include the application name. This means that the user needs to identify apps by its icon, which in many cases is the default placeholder. 

selection_729

The same concept should be applied to the service catalog listing that @andresmgot is working on. This should include the name of the service class.

In terms of its visualization, we might be able to add this app/instance class name before the chart version at the bottom left of the card.

### PR DESCRIPTION
**Pull request by andresmgot**
Fixes #764 

Add the chart name to the app list and the service name in the service instance list:

screenshot 2018-11-09 at 11 23 16
screenshot 2018-11-09 at 11 23 33

Note that changes in tiller-proxy were required in order to return the chart name in the list overview.

### CODE REVIEW

File: dashboard/src/components/AppList/AppListItem.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ class AppListItem extends React.Component<IAppListItemProps> {
         link={`/apps/ns/${app.namespace}/${app.releaseName}`}
         title={app.releaseName}
         icon={icon}
-        info={app.version || "-"}
+        info={`${app.chart}/v${app.version || "-"}`}
```
Comment (prydonius) [inline dashboard/src/components/AppList/AppListItem.tsx]: the / version is confusing, for a second I thought it was repo/name. Maybe we could have a space between these instead?

Comment (migmartri) [reply dashboard/src/components/AppList/AppListItem.tsx]: or maybe `colon` which seems more widespread? Alternative I'd go for the version in parenthesis

Comment (migmartri) [reply dashboard/src/components/AppList/AppListItem.tsx]: Why do we need `-`? `${app.version || "-"}`

File: dashboard/src/components/ServiceInstanceList/ServiceInstanceCardList.tsx [hunk 2]
```diff
@@ -45,6 +45,7 @@ const ServiceInstanceCardList: React.SFC<IServiceInstanceCardListProps> = props
                   namespace={instance.metadata.namespace}
                   link={link}
                   icon={icon}
+                  serviceClassName={(svcClass && svcClass.spec.externalName) || "-"}
```
Comment (migmartri) [inline dashboard/src/components/ServiceInstanceList/ServiceInstanceCardList.tsx]: wondering if to be safe you should also check `svcClass.spec`

File: dashboard/src/components/AppList/AppListItem.tsx (hunk 1 above)
Comment (prydonius) [reply dashboard/src/components/AppList/AppListItem.tsx]: I think we should avoid confusing this with the chart version, the Helm CLI uses `{chart.name}-{chart.version}`. IMO a space will be the least confusing.

Comment (andresmgot) [reply dashboard/src/components/AppList/AppListItem.tsx]: Actually I think that the separation between the two items using just a space is not very visible, let me try using two different lines

Comment (andresmgot) [reply dashboard/src/components/AppList/AppListItem.tsx]: nvm, I don't like the result of printing them as a column:
screenshot 2018-11-09 at 12 14 17

**Pull request merged**